src/math/hyp1f1.hh

```diff
@@ -396,16 +400,18 @@ namespace rascal {
 
       double z_asympt{1.};
 
-      size_t nterms_a{0}, nterms_s{0};
+      // TODO(all) TODO(felix) I remember this was a quick fix and I am not
```

Comment:
What exactly was a quick fix?

Reply:
If this is not needed anymore, remove it?